lib/linux_admin/fstab.rb

```diff
@@ -44,30 +44,31 @@ def columns
     end
 
     def column_lengths
-      self.columns.collect { |c| c ? c.size : 0 }
+      columns.collect { |c| c ? c.to_s.size : 0 }
     end
 
     def formatted_columns(max_lengths)
       self.columns.collect.
-        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ")
+        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ").rstrip
     end
   end
 
   class FSTab
     include Singleton
 
-    attr_accessor :entries
-    attr_accessor :maximum_column_lengths
```

Comment:
Maybe we should keep a method for this so we don't change API compatibility.

Reply:
Done